third_party/eigen3/gpu_packet_math.patch

```diff
@@ -4,15 +4,15 @@
    return make_double2(from, from);
  }
  
-+#if defined(EIGEN_CUDA_ARCH)
++#if defined(EIGEN_CUDA_ARCH) || defined(EIGEN_HIP_DEVICE_COMPILE)
```

Comment:
@rmlarsen 

this one change is not part of the Eigen PR I mentioned. That is because it modifies the existing content in TF eigen patch (as opposed to the other changes that only append content to the TF eigen patch).

So we will need to make this modification on your end. Also I do not know if you intend to upstream the TF eigen patch, if you do please include the HIP specific change here.

Thanks

deven

Reply:
Yeah, we need to upstream the last few bits in that patch file. I'll do it today or Monday.